src/main.ts

```diff
@@ -29,8 +32,17 @@ import { getCommentMessage } from "#/main/modules/get_comment_message";
 export const _main_ = async () => {
   // Verify correct environment and request context
   requireEvent();
-  requirePullNumber();
+  // requirePullNumber();  
```

Comment:
We should remove this if we don't want it.  Generally speaking, commented code shouldn't be left in the repository without a pretty compelling reason.